content/tags/tags.json

```diff
@@ -1,12 +1,20 @@
 [
 	{
-		"tag": "Silicon Alley",
```

Comment:
Can we add a tag for general tech?

Reply:
Yup, I'll add the tag when I put in an article with the tag. I didn't think it was necessary for this pr so I only included the default tags in the database.